### DISCUSSION TITLE
using custom ui for coolify

### DISCUSSION THREAD

**Discussion by twlite** [coollabsio/coolify#1703] [Q&A] [answered]
:wave: really nice project, thank you for building this. I was wondering if it is possible to plug custom ui instead of using the builtin one for coolify? In other words, using coolify's api with custom user interface to match my own design system? would that be possible?

It would also allow me to place custom features outside the scope of coolify for example.

**Comment by unpseudocomplique** [comment, marked as answer]
I think it will be possible to some extent.
There is an api and will gain features over time. We are already using the api to have a custom deploy page since we have specific need for deploy!
